charts/avatar/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: avatar-api
+name: avatar
```

Comment:
Why was this change made? I thought we agreed the endpoint would be called `avatar-api`

Reply:
Nikhil is making this chart work like tfr-backend where the api and worker are in the same helm chart. Because of this, it doesn't make sense to call the chart -api when it contains more than just an api.